arlo_server/routes.py

```diff
@@ -1088,6 +1091,18 @@ def audit_report(election_id):
     )
 
 
+# TODO make this work for multiple contests
+# TODO include comments
```

Comment:
will come back to this when i update the audit report for multi-jurisdiction in general